src/test/java/examples/ExamplesTest.java

```diff
@@ -0,0 +1,18 @@
+package examples;
```

Comment:
This looks like a runner file
Please rename as ExamplesTestSuiteRunner.java

Reply:
Updated